src/app/locales/pt-br/settings.yml

```diff
@@ -0,0 +1,571 @@
+# Translation notes:
+# I list some translation made here of some words not so used in portuguese
+# to make it clear and avoid confusion about it.
+#
+# tray (system tray): área de notificação
+# streaming: transmissão
+# buffering: bufferização
+# to launch (stream): abrir (transmissão)
```

Comment:
I've had the same issues while writing the German translations. I can't comment on how well your current translations fit into pt-br, but looking at the translations on Twitch's website for similar stuff could help maybe.

Reply:
> All good. Pull requests don't have to be perfect when they are submitted. I'm always happy to help and/or discuss things. Just curious, has there been anything in the wiki which was a bit unclear? That might be a reason to further improve it.
> https://github.com/streamlink/streamlink-twitch-gui/wiki/Translating

In order to add 'pt-br' translation I had to add the 'app/locales/**_pt_**/config.js' file. Otherwise ember cant find 'pt' namespace (I figured out this looking for chinese _zh_ translation). A small detail but it was not write in the document.

> Otherwise, thanks again for your PR. I'll leave it up for a bit for further review and/or further changes/improvements.

You are welcome. :)

Reply:
The `ember-i18n` lib that's currently being used here doesn't have configs for language codes with region data, which is the reason why you had to import the `pt` module instead (or write your own one):
https://github.com/jamesarosen/ember-i18n/tree/v5.2.0/addon/config

However, due to `ember-i18n` being discontinued in favor of `ember-intl`, I'll have to switch the translation libs soon. Since `ember-intl` uses native Intl-APIs and also a more standard syntax for pluralization rules in the translations, those custom locale config imports are not needed anymore, making everything a bit more simple.